src/content/en/fundamentals/primers/push/notifications.markdown

```diff
@@ -0,0 +1,82 @@
+---
+layout: shared/narrow
+title: "Notifications"
+description: "The notification is the actual message that is sent to a user after he has enabled push messages and has subscribed to the site. Messages are often sent via Google Cloud Messaging (GCM), a service that handles all aspects of message queueing and delivery to client applications running on target devices, but there are other services that support push notifications."
+authors:
+- dgash
+published_on: 2015-10-01
+updated_on: 2015-10-01
+order: 3
+---
+
+<p class="intro">The notification is the actual message that is sent to a user after he has enabled push messages and has subscribed to the site. Messages are often sent via <a href="https://developer.android.com/google/gcm/index.html">Google Cloud Messaging (GCM)</a>, a service that handles all aspects of message queueing and delivery to client applications running on target devices, but there are other services that support push notifications.
+</p>
+
+At the most basic level, push notifications from a web app need a back-end service to handle messaging. Chrome currently uses Google Cloud Messaging (GCM). While the eventual goal is for Chrome and GCM to support the [Web Push Protocol](https://datatracker.ietf.org/doc/draft-thomson-webpush-protocol/), other browsers are free to use other services.
+
+###Displaying
+
+To use the GCM API, you must set up a project in the Google Developer Console. Follow the instructions in the getting started guide for either [Android](https://developers.google.com/cloud-messaging/android/start) or [iOS](https://developers.google.com/cloud-messaging/ios/start). Make sure to enable "Google Cloud Messaging for Android", and make a note of the **project number** and **API key,** as you’ll need them later on. The project number is used in the Web App manifest (below) as the *gcm_sender_id* parameter, and you will need the Public API Keyon your server when you use the GCM API.
```

Comment:
For code snippets, use the \` symbol to wrap. For example: as the \`gcm_sender_id\`